code/modules/admin/verbs/mapping.dm

```diff
@@ -161,7 +161,7 @@ var/intercom_range_display_status = 0
 	src.verbs += /client/proc/Test_ZAS_Connection
 	src.verbs += /client/proc/SDQL2_query
 	src.verbs += /client/proc/check_sim_unsim
-	src.verbs += /proc/generateMiniMaps
+	//src.verbs += /proc/generateMiniMaps
```

Comment:
?

Reply:
you're right, might as well remove it now.